common/utilities.go

```diff
@@ -218,3 +218,12 @@ func GetSystemNetworkByName(network_list []networks.Network, network_name string
 	}
 	return nil
 }
+
+func GetSystemNetworkByType(network_list []networks.Network, network_type string) *networks.Network {
```

Comment:
Please add test case against the new method in utilities

Reply:
Done.